subprojects/core/src/main/java/org/gradle/api/internal/artifacts/dsl/dependencies/DefaultArtifactTransformTargets.java

```diff
@@ -20,16 +20,22 @@
 import org.gradle.api.artifacts.transform.ArtifactTransformTargets;
 import org.gradle.api.attributes.AttributeContainer;
 import org.gradle.api.internal.attributes.AttributeContainerInternal;
-import org.gradle.api.internal.attributes.DefaultAttributeContainer;
+import org.gradle.api.internal.attributes.DefaultMutableAttributeContainer;
+import org.gradle.api.internal.attributes.ImmutableAttributesFactory;
 
 import java.util.List;
 
 public class DefaultArtifactTransformTargets implements ArtifactTransformTargets {
+    private final ImmutableAttributesFactory cache;
 
```

Comment:
You're right, this is leftover from an initial implementation. I thought "factory" reflected the concept better than cache. I'll change those, thanks for spotting!